### PR TITLE
fix export zip for chrome

### PR DESCRIPTION
**Pull request by renchauret**

### CODE REVIEW

File: client/src/components/Export/ExportToZipButton/ExportToZipButton.js [hunk 1]
```diff
@@ -24,20 +24,30 @@ export const ExportToZipButton = (props) => {
...
+                        Bucket: 'monuments-and-memorials',
+                        Key: key
+                    }).promise();
+                    let name = getS3ImageNameFromObjectUrl(image.url)
+                    if (!name.endsWith('.png') && !name.endsWith('.jpg')) {
+                        name = name + '.jpg'
                     }
-                )
+                    zip.file(name, data.Body, {binary: true});
+                    console.log(data)
```
Comment (wrs1492) [inline client/src/components/Export/ExportToZipButton/ExportToZipButton.js:46]: Does this need to be here?

File: client/src/components/Export/ExportToZipButton/ExportToZipButton.js [hunk 2]
```diff
@@ -24,20 +24,30 @@ export const ExportToZipButton = (props) => {
...
+                } catch (e) {
```
Comment (wrs1492) [inline client/src/components/Export/ExportToZipButton/ExportToZipButton.js:47]: Do we want to do anything with the caught exception?

File: client/src/components/Export/ExportToZipButton/ExportToZipButton.js (hunk 1 above)
Comment (renchauret) [reply client/src/components/Export/ExportToZipButton/ExportToZipButton.js]: removed it

File: client/src/components/Export/ExportToZipButton/ExportToZipButton.js (hunk 2 above)
Comment (renchauret) [reply client/src/components/Export/ExportToZipButton/ExportToZipButton.js:46]: added rollbar logging

**Review by wrs1492 — APPROVED** [review]

**Pull request merged**